packages/ia-pic-uploader/src/ia-pic-uploader.ts

```diff
@@ -560,38 +571,66 @@ export class IAPicUploader extends LitElement {
           >
         </div>
       </div>
+      <span class="error">${this.fileValidationError}</span>
     `;
   }
 
   /**
    * function that render html for overlay form compact version
    * @returns {HTMLElement}
    */
-  get overlayIcon() {
+  get overlayTemplate() {
     return html`
       <div
-        class="overlay-icon ${this.showLoadingIndicator
+        class="overlay ${this.showLoadingIndicator
           ? 'show-overlay pointer-none'
           : ''}"
-        @keyup=""
+        @keyup="${() => {}}"
         @click=${() => {
           this.dropRegion?.click();
         }}
       >
-        ${this.showLoadingIndicator ? this.loadingIndicatorTemplate : '+'}
+        ${this.showLoadingIndicator
+          ? this.loadingIndicatorTemplate
+          : this.plusSVGTemplate(25, 25, '#fff', '#333333')}
       </div>
     `;
   }
 
+  /**
+   *  Render svg plus
+   * 
+   * @param {number} height | height for svg 
+   * @param {number} width  | width for svg
+   * @param {string} fill | fill color
+   * @param {string} stroke | stroke color
+   * @returns {SVGAElement}
+   */
+  plusSVGTemplate(height: number, width: number, fill: string, stroke: string) {
+    return svg`<svg 
```

Comment:
Note: if you're templating an entire `<svg>` element, it's recommended to still use Lit's `html` tag. The `svg` tag is for templating _inner fragments_ of an SVG. [See examples here](https://lit.dev/docs/api/templates/#svg).